modules/cbmarkdown/models/Processor.cfc

```diff
@@ -24,13 +15,13 @@ component accessors="true" singleton{
 	*/
 	function init( required javaLoader ){
 		// store references
-		variables.javaLoader = arguments.javaLoader;
-		// Setup markdown processor
-		variables.config = arguments.javaLoader.create( "com.github.rjeschke.txtmark.Configuration" )
-			.builder()
-			.forceExtentedProfile()
+		variables.jl = arguments.javaLoader;
```

Comment:
Don't use `jl` as a reference since that can mean nothing.  Use abbreviations only when using commonly know abbreviations or acronyms.  Call it as it is `javaloader`